.release-please-manifest.json

```diff
@@ -1 +1,8 @@
-{"packages/build":"18.9.0","packages/cache-utils":"2.0.3","packages/config":"15.6.2","packages/functions-utils":"2.0.7","packages/git-utils":"2.0.1","packages/run-utils":"2.0.1"}
\ No newline at end of file
+{
+  "packages/build": "18.9.0",
+  "packages/cache-utils": "2.0.3",
+  "packages/config": "15.6.2",
+  "packages/functions-utils": "2.0.7",
+  "packages/git-utils": "2.0.1",
+  "packages/run-utils": "2.0.1"
+}
```

Comment:
This change seems to be due to some update with `release-please` which conflicts with Prettier.
This is unrelated to this PR but is blocking it.